css/exihib-statment.css

```diff
@@ -18,6 +18,33 @@ ul#list-container li:hover {
   opacity: 0.6;
 }
 
+@media only screen and (max-width: 500px) {
+  .l-list1 {
+    margin: 0 20px 0 0;
+  }
+  .ex-statement-p,
+  a {
```

Comment:
@mohamedlafiteh could you please change this so that you are not using a generic a selector here? This style will be applied to all links on the page as it's not scoped in any way to just the statement.

Reply:
I removed the comma here as well.